server/build.gradle

```diff
@@ -69,7 +69,7 @@ dependencies {
   api project(":libs:opensearch-geo")
   api project(":libs:opensearch-telemetry")
   api project(":libs:opensearch-task-commons")
-
```

Comment:
I think I will remove this dependency for now as server isn't using it right now. I will add it in the next PR if we agree upon arrow being part of the core.